lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -3928,26 +4025,44 @@ prop_balanceTransactionValid
     -> ShowBuildable (PartialTx Cardano.AlonzoEra)
     -> StdGenSeed
     -> Property
-prop_balanceTransactionValid wallet (ShowBuildable partialTx) seed
+prop_balanceTransactionValid wallet@(Wallet' _ walletUTxO _) (ShowBuildable partialTx) seed
     = withMaxSuccess 1_000 $ do
         let combinedUTxO = mconcat
                 [ view #inputs partialTx
                 , Compatibility.toCardanoUTxO Cardano.ShelleyBasedEraAlonzo walletUTxO
                 ]
         let originalBalance = txBalance (view #tx partialTx) combinedUTxO
-        let res = balanceTransaction'
+        let res = balanceTx
                 wallet
+                (mockBundledProtocolParametersForBalancing Cardano.AlonzoEra)
+                dummyTimeInterpreter
                 seed
                 partialTx
-        case res of
+        let originalOuts = txOutputs (view #tx partialTx)
+
+        let classifications =
```

Comment:
Moving these outside the `Right` pattern match. Otherwise the results are heavily biased.